core/src/main/java/com/yandex/yoctodb/util/mutable/impl/AscendingBitSetIndexToIndexMultiMap.java

```diff
@@ -18,9 +18,8 @@
  * {@link IndexToIndexMultiMap} implementation based on grouped {@link LongArrayBitSet}s
  * For more information see {@link com.yandex.yoctodb.util.immutable.impl.AscendingBitSetIndexToIndexMultiMap}
  *
- * @author Andrey Korzinev (goodfella@yandex-team.ru)
+ * @author Andrey Korzinev (ya-goodfella@yandex.com)
```

Comment:
Could you please add yourself to `CONTRIBUTORS` file?

Reply:
Done. + Added license headers to the new files.